tests/TestyMcTestFace.cpp

```diff
@@ -18,8 +20,19 @@ class FrameIsdTest : public ::testing::Test {
       json jsonIsd = json::parse(isdFile);
       isd.clearAllParams();
       for (json::iterator it = jsonIsd.begin(); it != jsonIsd.end(); ++it) {
-         isd.addParam(it.key(), it.value().dump());
+         if (it.value().size() >1 ) {
+            std::vector<double> v = it.value();
+            for (int j=0;j < v.size(); j++) {
```

Comment:
I thought in the JSON library it is not necessary to cast this into a standard vector? https://github.com/nlohmann/json#stl-like-access